web/themes/custom/oc_bootstrap/css/style.css

```diff
@@ -990,3 +990,48 @@ article.teaser h2 a {
     width: 1000px;
   }
 }
+
+.division-building-materials-amp-construction,
+.division-consumer-electronics,
+.division-fashion,
+.division-fmcg-amp-cpg,
+.division-industrial-equipment-amp-packaging,
+.division-paper-publishing-amp-printing,
+.division-pharmaceutical,
+.division-transportation-amp-automotive {
```

Comment:
I would wrap this into one class that gets applied to ever row. So your row class becomes `division-row division-{{ title | clean_class }}`

This rules then just needs to select `.division-row`